### ISSUE TITLE
[Bug Report] Getting `ModuleNotFoundError` when running `Grokking_Demo.ipynb`

### ISSUE THREAD

**Issue by anthonyduong9** [TransformerLensOrg/TransformerLens#621]
**Describe the bug**
I'm getting a `ModuleNotFoundError` when running `Grokking_Demo.ipynb`.

**Code example**
```
%pip install git+https://github.com/TransformerLensOrg/neel-plotly.git
from neel_plotly.plot import line
```

Error message: `ModuleNotFoundError: No module named 'neel_plotly'`

**System Info**
Describe the characteristic of your environment:
 * `transformer_lens` was installed with `pip`
 * MacOS
 * Python 3.11

**Additional context**
We replaced `neelnanda-io` with `TransformerLensOrgs` in https://github.com/TransformerLensOrg/TransformerLens/pull/580. Presumably, you'll encounter this bug if you run other demos that install from repos that TransformerLensOrg doesn't have.

### Checklist

- [ ] I have checked that there is no similar [issue](https://github.com/TransformerLensOrg/TransformerLens/issues) in the repo (**required**)

**Comment by anthonyduong9** [comment]
@bryce13950 I fixed this locally by replacing `TransformerLensOrg` with `neelnanda-io`. Do you think I should open a PR to fix it throughout the code by doing the same, or we should add the repos to TransformerLensOrg?

**Comment by bryce13950** [comment]
That is correct. TransformerLens moved to its current org, and we did a big find an replace at the time for `neelnanda-io` to `TransformerLensOrg`. Didn't think about the references to `neel_plotly` in the demos. If you can replace those in the rest of the demos, that would be super helpful.
